meilisearch-core/src/store/mod.rs

```diff
@@ -45,7 +45,7 @@ use crate::settings::SettingsUpdate;
 use crate::{query_builder::QueryBuilder, update, DocIndex, DocumentId, Error, MResult};
 
 type BEU64 = zerocopy::U64<byteorder::BigEndian>;
-type BEU16 = zerocopy::U16<byteorder::BigEndian>;
+pub type BEU16 = zerocopy::U16<byteorder::BigEndian>;
```

Comment:
This isn't necessary as this is used in the same crate and the `type` is at the top of the lib.

Reply:
```
rror[E0603]: type alias `BEU16` is private                                                                                                                                   
  --> meilisearch-core/src/facets.rs:17:19                                                                                                                                    
   |                                                                                                                                                                          
17 | use crate::store::BEU16;                                                                                                                                                 
   |                   ^^^^^ this type alias is private                                                                                                                       
   |                                                                                                                                                                          
note: the type alias `BEU16` is defined here                                                                                                                                  
  --> meilisearch-core/src/store/mod.rs:48:1                                                                                                                                  
   |                                                                                                                                                                          
48 | type BEU16 = zerocopy::U16<byteorder::BigEndian>;                                                                                                                        
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^     
```

Reply:
Ho, sorry this is because is is declared inside of store.rs and not lib.rs, my bad!